tests/js/common/shell/shell-index.js

```diff
@@ -1196,8 +1197,12 @@ function parallelIndexSuite() {
     },
 
     testCreateInParallel: function () {
-      let n = 80;
-      for (let i = 0; i < n; ++i) {
+      let noIndices = 80;
```

Comment:
Ok, `noIndices` actually doesn't mean `no indexes`, but it means `number of indexes`... So the variable name is at least confusing. When trying to find a prefix for "number" I suggest to use `nr`, `num` or just `number` instead of `no` in the future. It can stay like this in this PR.